java_collections_block/optional_tasks/src/main/resources/Output_file_for_task1.txt

```diff
@@ -0,0 +1,4 @@
+fourth line
```

Comment:
will this file be automatically generated during your program execution?

if so you should not comment it to the source code

Reply:
delete